frontend/src/layout/index.tsx

```diff
@@ -1,4 +1,4 @@
 // This funky expressions just means: above a screen width of 1100px, the extra
-// space will be 10% margin left and right. This is the middle ground between
+// space will be 3% margin left and right. This is the middle ground between
 // filling the full screen and having a fixed max width.
-export const OUTER_CONTAINER_MARGIN = "0 calc(max(0px, 100% - 1100px) * 0.1)";
+export const OUTER_CONTAINER_MARGIN = "0 calc(max(0px, 100% - 1100px) * 0.03)";
```

Comment:
Idk, it's according to the given design. I can revert this if you still prefer the old one.

Reply:
I'm a bit confused by the width of the design. So according to Sigma it's 1440 px wide, which is narrower than my logical 1920px screen. On my screen I think the design as is is too wide. I think we can get the correct wideness of Lisa's design by instead tweaking the 1100px value here. Or a mix of the twoo. I think 1200px and 0.1 should work? In Lisa's design we have 24px of margin on a 1440px wide screen. 1440 - 1200 = 240, 10% of which are 24, right?

Reply:
This current PR at 1920 wide:

![image](https://user-images.githubusercontent.com/7419664/221616165-5ff634c9-ae44-492d-9975-24cdca0decb7.png)

Reply:
> In Lisa's design we have 24px of margin on a 1440px wide screen. 1440 - 1200 = 240, 10% of which are 24, right?

Yes but the there is also a fixed padding of 16px, which means the actual margin would need to be 8px on a 1440px screen to match the design. Of course I could adjust your calculation accordingly, but then it would be too wide again on 1920px screens I think (less then previously in this PR but still more so than with your above calculation). So maybe the padding also needs to be responsive. Hm, I'll try to figure something out.

Reply:
oh good point. But then It would just need to be... 1360 right? 1440-1360 = 80, 10% of which are 8. But I'm also not married to Lisas design exactly. So, something like `calc(max(0px, 100% - 1200px) * 0.08)` would probably be fine with me (without having tested it :D)

Reply:
> But then It would just need to be... 1360 right? 1440-1360 = 80, 10% of which are 8.

Yep, that's what I meant by adjusting your calculation. I have already done so but wasn't sure if this would again be considered to wide on larger screens. As a reference, here's a screenshot (this is with -1360):
![Screenshot 2023-02-27 at 18-09-37 Tobira Videoportal](https://user-images.githubusercontent.com/94838646/221639776-0dbf8097-da76-4f86-bc47-8b3ed22db1ad.png)